docker-compose.dev.yml

```diff
@@ -1,31 +1,65 @@
 version: "3"
 services:
-  bassa-api:
+  api:
     build: components/core/
+    container_name: api
     ports:
       - 5000:5000
+    links:
+      - db
+    volumes:
+      - ./downloads:/downloads
+    environment:
+      BASSA_DB_NAME: Bassa
+      BASSA_DB_USERNAME: bassa
+      BASSA_DB_PASSWORD: bassa@1234
     labels:
-      com.example.description: "Docker container for Bassa UI"
```

Comment:
I wonder if we should change these to `org.scorelab.bassa`?  What do @vivonk, @agentmilindu think?  🤔

Reply:
@JaDogg you are right bro. 
I will make a separate PR for labels as I think I have to change that in many places.
Please shall we keep that for now?

Reply:
Sure :)